docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/FeatureSelection/SelectFeaturesBasedOnCount.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic
+{
+    public static class SelectFeaturesBasedOnCount
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
+            var rawData = GetData();
+            var data = mlContext.Data.LoadFromEnumerable(rawData);
+
+            var convertedData = mlContext.Data.CreateEnumerable<InputData>(data, true);
+
+            Console.WriteLine("Contents of two columns 'NumericVector' and 'StringVector'.");
+            foreach (var item in convertedData)
+                Console.WriteLine("{0}\t\t\t{1}", string.Join("\t", item.NumericVector), string.Join("\t", item.StringVector));
+            // 4       NaN     6                       A       WA   Male
+            // 4       5       6                       A            Female
```

Comment:
>      [](start = 63, length = 5)

align for just here.. no need to make it match exactly to the output and have it looked tabbing off.

Reply:
or is it a separate column?

It helps to print the headers. 

---
In reply to: [272391190](https://github.com/dotnet/machinelearning/pull/3184#discussion_r272391190) [](ancestors = 272391190)

Reply:
the alignment here seems good to me... for text this slot is empty or null .. 

not sure if u meant something else

---
In reply to: [272391190](https://github.com/dotnet/machinelearning/pull/3184#discussion_r272391190) [](ancestors = 272391190)